app/uk/gov/hmrc/servicecommissioningstatus/service/StatusCheckService.scala

```diff
@@ -32,7 +32,10 @@ class StatusCheckService @Inject()(
   releasesConnector       : ReleasesConnector
 )(implicit ec: ExecutionContext){
 
-  def commissioningStatusChecks(serviceName: String)(implicit hc: HeaderCarrier): Future[ServiceCommissioningStatus] =
+  def commissioningStatusChecks(serviceName: String): Future[ServiceCommissioningStatus] = {
+
+    implicit val hc: HeaderCarrier = HeaderCarrier()
```

Comment:
This should be provided from the `Controller` capturing the caller's context. E.g. `Controller with FrontendHeaderCarrierProvider`